runner/package.json

```diff
@@ -16,18 +16,8 @@
     "@double-agent/collect-controller": "1.0.0",
     "@double-agent/real-user-agents": "1.0.0",
     "@double-agent/config": "1.0.0",
-    "secret-agent": "^1.4.1-alpha.5",
-    "@secret-agent/core": "^1.4.1-alpha.5",
-    "@secret-agent/emulate-chrome-ancient": "^1.4.1-alpha.5",
-    "@secret-agent/emulate-chrome-80": "^1.4.1-alpha.5",
-    "@secret-agent/emulate-chrome-81": "1.4.1-alpha.5",
-    "@secret-agent/emulate-chrome-83": "^1.4.1-alpha.5",
-    "@secret-agent/emulate-chrome-84": "^1.4.1-alpha.5",
-    "@secret-agent/emulate-chrome-85": "1.4.1-alpha.5",
-    "@secret-agent/emulate-chrome-86": "^1.4.1-alpha.5",
-    "@secret-agent/emulate-chrome-87": "^1.4.1-alpha.5",
-    "@secret-agent/emulate-chrome-88": "1.4.1-alpha.5",
-    "@secret-agent/emulate-safari-13": "^1.4.1-alpha.5",
+    "@secret-agent/core": "^1.4.1-alpha.4",
```

Comment:
Doesn't this mean we're only running double-agent tests on a single engine now? Wouldn't that make it fail most of the tests?

Reply:
Great question. I'm not sure.